lib/model/autocomplete.dart

```diff
@@ -145,6 +145,18 @@ class AutocompleteViewManager {
     autocompleteDataCache.invalidateUser(event.userId);
   }
 
+  void handleMessageEvent(MessageEvent event) {
+    for (final view in _mentionAutocompleteViews) {
+      view.refreshStaleUserResults();
+    }
```

Comment:
Wouldn't it be good to extract this bit to a (private) method as it is used in several other neighboring methods?! A name like `refreshAutocompleteResults` may be a proper name for that method as then we can replace it with all the method calls that just have the body of this method in their bodies.

Reply:
Yeah, a prep commit to factor this out would be good. A good name would be `_refreshStaleUserResults`, to keep it more specific. (Right now the only kind of autocomplete we have is to return users, but we'll also have autocomplete for streams and topics and other things, and those won't necessarily need to get invalidated when the list of users or their ranking changes.)